admin/config/environment.js

```diff
@@ -45,6 +45,7 @@ module.exports = function(environment) {
         NOT_FOUND: '404',
       },
       MAX_CONCURRENT_AJAX_CALLS: _getEnvironmentVariableAsNumber({ environmentVariableName: 'MAX_CONCURRENT_AJAX_CALLS', defaultValue: 8, minValue: 1 }),
+      METABASE_ORGANIZATION_DETAIL_DASHBOARD_URL: process.env.METABASE_ORGANIZATION_DETAIL_DASHBOARD_URL || 'https://metabase.pix.fr/dashboard/137?id=',
```

Comment:
Je propose de ne pas mettre de valeur par défaut ici, et de ne pas afficher le lien si la connexion à un tableau de bord externe n'est pas activée par la présence de cette variable d'environnement.

Je propose également de ne pas faire apparaître le nom `METABASE` dans cette URL qui associe inutilement un outil en particulier à cette fonctionnalité.

```suggestion
      ORGANIZATION_DASHBOARD_URL: process.env.ORGANIZATION_DASHBOARD_URL,
```

Reply:
Corrigé